test/e2e/v1beta1/scripts/gh-actions/build-load.sh

```diff
@@ -48,14 +51,7 @@ _build_containers() {
   done
 
   echo -e "\nBuilding $CONTAINER_NAME image with $DOCKERFILE...\n"
-  docker buildx build --platform "$(uname -m)" --load -t "$REGISTRY/$CONTAINER_NAME:$TAG" -f "../../../../../$DOCKERFILE" ../../../../../
-}
-
-_load_minikube_cluster() {
-  CONTAINER_NAME=${1:-"katib-controller"}
-
-  echo -e "\n\nLoading $CONTAINER_NAME image...\n\n"
-  minikube image load "$REGISTRY/$CONTAINER_NAME:$TAG"
+  DOCKER_BUILDKIT=1 minikube image build --build-opt platform=linux/amd64 --all -t "$REGISTRY/$CONTAINER_NAME:$TAG" -f "$DOCKERFILE" .
```

Comment:
To avoid OOM errors in CI, we must build container images using `minikube` instead of `docker`.

Reply:
What is the difference here?

Reply:
The build succeeded but tests take longer time

Reply:
@johnugeorge In `minikube image build`, we can build images using an actual container runtime on the minikube cluster. So we no longer load images to the minikube cluster.

Reply:
> The build succeeded but tests take longer time

@johnugeorge  As I can see, It seems to be shorter.

<img width="1396" alt="2023-01-17 3 13 46" src="https://user-images.githubusercontent.com/68272500/212743515-0ede799b-0970-4db1-bc31-d4bdc11c8f11.png">